app/src/main/java/com/example/hungrywolfes/ui/MainActivity.kt

```diff
@@ -0,0 +1,43 @@
+package com.example.hungrywolfes.ui
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.view.View
+import androidx.navigation.NavController
+import androidx.navigation.fragment.NavHostFragment
+import androidx.navigation.ui.setupWithNavController
+import com.example.hungrywolfes.R
+import com.google.android.material.bottomnavigation.BottomNavigationView
+
+
+class MainActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_main)
+
+        val navHostFragment =
+            supportFragmentManager.findFragmentById(R.id.nav_host_fragment) as NavHostFragment
+
+
+        val navController = navHostFragment.navController
+        navController.addOnDestinationChangedListener(destinationChangedListener)
+
+
+        findViewById<BottomNavigationView>(R.id.bottom_navigation)?.setupWithNavController(
```

Comment:
You could have used binding here. 
https://developer.android.com/topic/libraries/data-binding/expressions#binding_data